solidity/test/helpers/Constants.js

```diff
@@ -1,27 +1,31 @@
-const { web3 } = require('@openzeppelin/test-environment');
+const { BigNumber, ethers } = require('ethers');
 
 module.exports = {
     registry: {
-        CONTRACT_REGISTRY: web3.utils.asciiToHex('ContractRegistry'),
-        BANCOR_NETWORK: web3.utils.asciiToHex('BancorNetwork'),
-        NETWORK_SETTINGS: web3.utils.asciiToHex('NetworkSettings'),
-        CONVERTER_FACTORY: web3.utils.asciiToHex('ConverterFactory'),
-        CONVERSION_PATH_FINDER: web3.utils.asciiToHex('ConversionPathFinder'),
-        CONVERTER_UPGRADER: web3.utils.asciiToHex('BancorConverterUpgrader'),
-        CONVERTER_REGISTRY: web3.utils.asciiToHex('BancorConverterRegistry'),
-        CONVERTER_REGISTRY_DATA: web3.utils.asciiToHex('BancorConverterRegistryData'),
-        BNT_TOKEN: web3.utils.asciiToHex('BNTToken'),
-        BANCOR_X: web3.utils.asciiToHex('BancorX'),
-        BANCOR_X_UPGRADER: web3.utils.asciiToHex('BancorXUpgrader'),
-        NETWORK_SETTINGS: web3.utils.asciiToHex('NetworkSettings')
+        CONTRACT_REGISTRY: ethers.utils.formatBytes32String('ContractRegistry'),
+        BANCOR_NETWORK: ethers.utils.formatBytes32String('BancorNetwork'),
+        NETWORK_SETTINGS: ethers.utils.formatBytes32String('NetworkSettings'),
+        CONVERTER_FACTORY: ethers.utils.formatBytes32String('ConverterFactory'),
+        CONVERSION_PATH_FINDER: ethers.utils.formatBytes32String('ConversionPathFinder'),
+        CONVERTER_UPGRADER: ethers.utils.formatBytes32String('BancorConverterUpgrader'),
+        CONVERTER_REGISTRY: ethers.utils.formatBytes32String('BancorConverterRegistry'),
+        CONVERTER_REGISTRY_DATA: ethers.utils.formatBytes32String('BancorConverterRegistryData'),
+        BNT_TOKEN: ethers.utils.formatBytes32String('BNTToken'),
+        BANCOR_X: ethers.utils.formatBytes32String('BancorX'),
+        BANCOR_X_UPGRADER: ethers.utils.formatBytes32String('BancorXUpgrader'),
+
+        //
+        BANCOR_FORMULA: ethers.utils.formatBytes32String('BancorFormula')
     },
 
     roles: {
-        ROLE_OWNER: web3.utils.keccak256('ROLE_OWNER'),
-        ROLE_GOVERNOR: web3.utils.keccak256('ROLE_GOVERNOR'),
-        ROLE_MINTER: web3.utils.keccak256('ROLE_MINTER'),
-        ROLE_SEEDER: web3.utils.keccak256('ROLE_SEEDER')
+        ROLE_OWNER: ethers.utils.id('ROLE_OWNER'),
+        ROLE_GOVERNOR: ethers.utils.id('ROLE_GOVERNOR'),
+        ROLE_MINTER: ethers.utils.id('ROLE_MINTER'),
+        ROLE_SEEDER: ethers.utils.id('ROLE_SEEDER')
     },
 
-    NATIVE_TOKEN_ADDRESS: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
+    NATIVE_TOKEN_ADDRESS: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
+    MAX_UINT256: BigNumber.from('2').pow(BigNumber.from('256')).sub(BigNumber.from('1')),
+    ZERO_ADDRESS: ethers.constants.AddressZero
```

Comment:
I'm not sure about it, but perhaps it'd better to use `const { AddressZero } = ethers.constants` in the code instead of exposing it? What do you think?

Reply:
We could but it just feel easier to just have to import something as a whole as we usually need several Constants in one file. Feels cleaner to me on the importer file - imo

Reply:
Yep, I agree.